### ISSUE TITLE
Confusion with conv1 accepting x and task_label in _forward_impl method of the 'ResNetGrad' class in models/vlcs.py

### ISSUE THREAD

**Issue by ZexiChen502** [MediaBrain-SJTU/MoLA#2]
I encountered an confusion in the _forward_impl method of the ResNetGrad class in models/vlcs.py, where conv1 is called with both x and task_label. Since conv1 is a standard torch.nn.Conv2d layer, it can only accept a single input tensor x. Passing two arguments (x and None) may result in an error?

class ResNetGrad(nn.Module):

    def __init__(self, block, layers, num_classes=1000, zero_init_residual=False, groups=1, width_per_group=64,
                 replace_stride_with_dilation=None, norm_layer=None, ratio=1, lora_layer=None, lora_rank=None, **kwargs):
        super(ResNetGrad, self).__init__()
        ......
        self.conv1 = torch.nn.Conv2d(3, self.inplanes, kernel_size=7, stride=2, padding=3, bias=False)
        ......

    def _forward_impl(self, x, task_label=None, return_features=False):
       ......
        x = self.conv1(x, task_label)
       ......
Great work, looking forward to your team's reply
